CONTRIBUTORS.yaml

```diff
@@ -214,6 +214,10 @@ bfranicevic:
     name: Branka Franicevic
     joined: 2021-10
 
+breakthewall:
+    name: Joan Hérisson
+    email: joan.herisson@univ-evry.fr
```

Comment:
```suggestion
    email: joan.herisson@univ-evry.fr
    joined: 2022-03
```

Reply:
could you add a "joined" line like this to all the new contributors you added? :)

Reply:
ok done. I also added 2 more contributors.